### ISSUE TITLE
No option to Request for Code and Token or Token alone

### ISSUE THREAD

**Issue by achyuthgroot** [auth0/auth0-react#188]
Seems like there is no option to request for code and token together.

I'm not sure how to categorize this scenario exactly. 

Previously I have used few openid plugins where I can initiate an authentication request with both code and token together. During redirection we will receive both code and token as per the response mode.

Is there any option available for this in this plugin ? If Yes please specify that.

My requirement is something like below:

`<Auth0Provider
        domain="DOMAIN"
        clientId="CLIENT_ID"
        redirectUri="URL"
        response_type="code token"
    ></Auth0Provider>`

Basically I would like to add another option for `response_type` or `responseType` where I can pass wether I need token or code or both together.

Any help would really appreciated

Thanks in advance.

**Comment by adamjmcgrath** [comment]
Hi @achyuthgroot - thanks for raising this

This SDK uses the code flow with PKCE only - we don;t recommend using Implicit (or in your case Hybrid) flows for SPA applications.

More info here https://auth0.com/blog/oauth2-implicit-grant-and-spa/

**Comment by fifithecat** [comment]
Good to hear the confirmation, it seems the documentation is not very clear about whether this package is using implicit flow or PKCE
